gpdb-doc/dita/ref_guide/config_params/guc-list.xml

```diff
@@ -471,6 +471,9 @@
             </li>
             <li><xref href="#optimizer_enable_orderedagg" type="section"
                 >optimizer_enable_orderedagg</xref></li>
+            <li>
+              <xref href="#optimizer_enable_table_alias" format="dita"
```

Comment:
What is mean `format="dita"`?

Reply:
It seems to be yet another one documentation format. 
https://github.com/arenadata/gpdb/blob/2fd27bbc76ae4547cd99d1edac260805f6655cd6/gpdb-doc/README.md?plain=1#L5
I've just copied it from another topic in the same document.

Reply:
why not `type="section"` as above and below?

Reply:
because if was copy-pasted in other place, where "dita" was above and below and then placed alphabetically